tools/fpgainfo/board.c

```diff
@@ -66,6 +66,7 @@ static platform_data platform_data_table[] = {
 	{ 0x8086, 0x0b2c, "libboard_d5005.so", NULL },
 	{ 0x8086, 0xaf00, "libboard_d5005.so", NULL },
 	{ 0x8086, 0xbcce, "libboard_d5005.so", NULL },
```

Comment:
What about this one (0x8086, 0xbcce)?  Does it need to be removed?

Reply:
0x8086, 0xbcce -> IOFS R1 on D5005 card
0x1c2c, 0xbcce -> next gen N6010 card , vender id is different(silicom)